xbmc/interfaces/legacy/Player.cpp

```diff
@@ -397,6 +404,20 @@ namespace XBMCAddon
       CServiceBroker::GetGUI()->GetWindowManager().SendMessage(msg);
     }
 
+    InfoTagGame* Player::getGameInfoTag()
+    {
+      XBMC_TRACE;
+      if (!g_application.GetAppPlayer().IsPlayingGame())
+        throw PlayerException("XBMC is not playing any game file");
```

Comment:
Should probably change XBMC -> KODI

Reply:
If you want it 100% correct, you want it to be `CCompileInfo::GetAppName()`